### ISSUE TITLE
Using Images

### ISSUE THREAD

**Issue by ezimerman** [Illexpert/illexpert.com#19]
Can you get your cards to populate self-hosted images, or is there a way to include alt text in the soundcloud hosted images? 

- Add image alt text
- Change image file names to reflect content

Do you have Instagram & Pinterest?

**Comment by Illexpert** [comment]
Yes, it's all a matter of engineering. I can get the cards to do whatever I want them to do, literally.
But the hours involved need to be taken into account.
We should go through the source code. That'll give you a better sense of what the engineering effort is to get different aspects complete. 
Also I'll discuss why I chose the implementation I did, that is a heavy reliance on the SoundCloud player-widget. (I tried alternatives, but building a robust sound widget takes many hours of dedication.) SoundCloud's is super for the effort involved.
